data/abilities.ts

```diff
@@ -397,13 +397,18 @@ export const Abilities: {[abilityid: string]: AbilityData} = {
 			return true;
 		},
 		onAfterMoveSecondary(target, source, move) {
+			const healingItems = [
```

Comment:
It seems like this is the third use of the same `healingItems` array in this file. Maybe it should be a file-level constant?